apps/ui/src/networks/offchain/api/index.ts

```diff
@@ -207,12 +207,12 @@ function formatVote(vote: ApiVote): Vote {
   };
 }
 
-async function formatUser(user: User) {
+function formatUser(user: User) {
```

Comment:
Can we remove this helper entirely and just set counts and name in `loadUser`?

The types here are not accurate anyway and it's only used in one place.

Reply:
Done ☑️